Objects/descrobject.c

```diff
@@ -1566,7 +1587,12 @@ property_descr_get(PyObject *self, PyObject *obj, PyObject *type)
 
     propertyobject *gs = (propertyobject *)self;
     if (gs->prop_get == NULL) {
-        PyErr_SetString(PyExc_AttributeError, "unreadable attribute");
+        if (gs->prop_name != NULL) {
+            PyErr_Format(PyExc_AttributeError, "unreadable attribute %S", gs->prop_name);
```

Comment:
These should use ``%R`` instead of ``%S``.

Reply:
Fixed